src/rpc/blockchain.cpp

```diff
@@ -499,14 +502,25 @@ UniValue MempoolToJSON(const CTxMemPool& pool, bool verbose)
         }
         return o;
     } else {
+        uint64_t mempool_sequence;
         std::vector<uint256> vtxid;
-        pool.queryHashes(vtxid);
-
+        {
+            LOCK(pool.cs);
+            pool.queryHashes(vtxid);
+            mempool_sequence = pool.GetSequence();
+        }
         UniValue a(UniValue::VARR);
         for (const uint256& hash : vtxid)
             a.push_back(hash.ToString());
 
-        return a;
+        if (!include_mempool_sequence) {
+            return a;
+        } else {
+            UniValue o(UniValue::VOBJ);
+            o.pushKV("txids", a);
+            o.pushKV("mempool_sequence", mempool_sequence);
```

Comment:
The behavior of `getrawmempool` with the additional param and its interaction with `verbose` should be covered in a functional test, unless I missed the added assertion somewhere.

Reply:
There is a test line for it:
`assert_raises_rpc_error(-8, "Verbose results can not contain mempool sequence values.", self.nodes[0].getrawmempool, True, True)`

Reply:
Oops, I even tested it manually as well. More like an `assert("mempool_sequence" in getrawmempool.keys())` or even checking the value too.

Reply:
Never mind, it looks like you covered that.